DeepSpeech.ipynb

```diff
@@ -172,7 +172,7 @@
     "export_version = 1\n",
     "remove_export = bool(int(os.environ.get('ds_remove_export', 0)))\n",
     "use_warpctc = bool(len(os.environ.get('ds_use_warpctc', '')))\n",
-    "default_stddev = float(os.environ.get('ds_default_stddev', 0.1))\n",
+    "default_stddev = float(os.environ.get('ds_default_stddev', 0.046875))\n",
```

Comment:
Interesting, I didn't know that.

However, what I'm doing is approximate xavier initialization.

I tried standard xavier initialization and the CTC algorithm ran into that problem again where it was unable to find the paths. So, I had to decrease the std from standard xavier initialization by a factor of 2.